asllib/doc/ASLSemanticsReference.tex

```diff
@@ -2164,9 +4333,15 @@ \section{SemanticsRule.SCall \label{sec:SemanticsRule.SCall}}
   \VerbatimInput[firstline=\SCallBegin, lastline=\SCallEnd]{../Interpreter.ml}
 
 \begin{emptyformal}
-  \subsection{Formally: sequential case}
-
-  \subsection{Formally: concurrent case}
+  \subsection{Formally}
+\begin{mathpar}
+  \inferrule{
+    \evalcall{\env, \name, \args, \namedargs} \evalarrow \Normal(\vg, \envp) \OrAbnormal
+  }
+  {
+    \evalstmt{\env, \texttt{S\_Call}(\name, \args, \namedargs)} \evalarrow \Continuing(\vg, \envp)
+  }
+\end{mathpar}
 \end{emptyformal}
 
 \isempty{\subsection{Comments}}
```

Comment:
From Semantics.SCond up to Semantics.SDeclNone - the table of contents is missing a space between the section number and the title itself, so that it looks like:
11.10SemanticsRule.SCond
11.11SemanticsRule.SCase 
[...]
11.20SemanticsRule.SDeclNone